### PR TITLE
Fixes civilian bounty job assignments, and lowers cooldown

### PR DESCRIPTION
**Pull request by KoboldCommando**
# About The Pull Request

Civilian bounties weren't responding properly to the assigned jobs.  They should provide two department-related bounties and one general bounty, but they were displaying totally random bounties.  This is fixed.

I also got a few complaints about the 5 minute cooldown between bounty picks, and I couldn't really think of a reason now to lower it, so it's not 60 seconds.  Enough time to hang out and probably socialize!

:cl:
fix: civilian bounties will now offer two department related bounties and one general bounty
balance: the cooldown for repicking bounties is now 60 seconds (down from 5 minutes)
/:cl:

### CODE REVIEW

File: code/modules/cargo/bounty.dm [hunk 1]
```diff
@@ -74,8 +74,13 @@ GLOBAL_LIST_EMPTY(bounties_list)
 	return TRUE
 
 // Returns a new bounty of random type, but does not add it to GLOB.bounties_list.
-/proc/random_bounty()
-	switch(rand(1, 13))
+/proc/random_bounty(guided = 0)
```
Comment (nednaZ) [inline code/modules/cargo/bounty.dm:77]: Default value would be invalid there, so just initialize as a null value
```suggestion
/proc/random_bounty(guided)
```

File: code/modules/cargo/bounty.dm [hunk 2]
```diff
@@ -100,21 +105,21 @@ GLOBAL_LIST_EMPTY(bounties_list)
 			var/subtype = pick(subtypesof(/datum/bounty/virus))
 			return new subtype
 		if(8)
-			var/subtype = pick(subtypesof(/datum/bounty/item/science))
-			return new subtype
-		if(9)
+			if(rand(2) == 1)
```
Comment (nednaZ) [inline code/modules/cargo/bounty.dm:108]: Better way to do a 50/50
```suggestion
			if(prob(50))
```

File: code/modules/cargo/bounty.dm [hunk 3]
```diff
@@ -74,8 +74,13 @@ GLOBAL_LIST_EMPTY(bounties_list)
...
-/proc/random_bounty()
-	switch(rand(1, 13))
+/proc/random_bounty(guided = 0)
+	var/bounty_num
+	if(guided && (guided != CIV_JOB_RANDOM))
+		bounty_num = guided
+	else
+		bounty_num = rand(1,12)
+	switch(bounty_num)
 		if(1)
```
Comment (nednaZ) [inline code/modules/cargo/bounty.dm:84]: I'd like these magic numbers all changed to their respective defines.
```suggestion
		if(CIV_JOB_BASIC)
```

File: code/modules/cargo/bounty.dm [hunk 4]
```diff
@@ -100,21 +105,21 @@ GLOBAL_LIST_EMPTY(bounties_list)
...
-			var/subtype = pick(subtypesof(/datum/bounty/item/science))
-			return new subtype
-		if(9)
+			if(rand(2) == 1)
+				var/subtype = pick(subtypesof(/datum/bounty/item/science))
+				return new subtype
 			var/subtype = pick(subtypesof(/datum/bounty/item/slime))
 			return new subtype
-		if(10)
+		if(9)
```
Comment (nednaZ) [inline code/modules/cargo/bounty.dm:113]:
```suggestion
		if(CIV_JOB_ENG)
```

File: code/modules/cargo/bounty.dm [hunk 5]
```diff
@@ -100,21 +105,21 @@ GLOBAL_LIST_EMPTY(bounties_list)
...
 			var/subtype = pick(subtypesof(/datum/bounty/item/engineering))
 			return new subtype
-		if(11)
+		if(10)
```
Comment (nednaZ) [inline code/modules/cargo/bounty.dm:116]:
```suggestion
		if(CIV_JOB_MINE)
```

File: code/modules/cargo/bounty.dm [hunk 6]
```diff
@@ -100,21 +105,21 @@ GLOBAL_LIST_EMPTY(bounties_list)
...
 			var/subtype = pick(subtypesof(/datum/bounty/item/mining))
 			return new subtype
-		if(12)
+		if(11)
```
Comment (nednaZ) [inline code/modules/cargo/bounty.dm:119]:
```suggestion
		if(CIV_JOB_MED)
```

File: code/modules/cargo/bounty.dm [hunk 7]
```diff
@@ -100,21 +105,21 @@ GLOBAL_LIST_EMPTY(bounties_list)
...
 			var/subtype = pick(subtypesof(/datum/bounty/item/medical))
 			return new subtype
-		if(13)
+		if(12)
```
Comment (nednaZ) [inline code/modules/cargo/bounty.dm:122]:
```suggestion
		if(CIV_JOB_GROW)
```

And so on. I can't do review actions on the other ones here.

**Review by nednaZ — CHANGES_REQUESTED** [review]
It looks like it will work as intended now, just need those changes.

**Comment by KoboldCommando** [comment]
All changes applied!

**Review by nednaZ — APPROVED** [review]
Looks good to me.

**Pull request merged**